Cargo.toml

```diff
@@ -1,6 +1,8 @@
 [package]
 name = "momento"
+# x-release-please-start-version
 version = "0.1.0"
+# x-release-please-end
```

Comment:
Seemed like Rust provides manifest version number as an [env var](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates) that's used when sending the `agent` header.
Put these comments here to get release-please to update version number in this file.

Reply:
sorry, not quite following the part about the env var, does that get used here somehow?

Reply:
So I think the utils file pulls in the [`CARGO_PKG_VERSION` env var](https://github.com/momentohq/client-sdk-rust/blob/main/src/utils.rs#L13) to use in the [`agent` header](https://github.com/momentohq/client-sdk-rust/blob/main/src/utils.rs#L104). And my understanding is that `CARGO_PKG_VERSION` gets pulled from this `.toml` file so I figured I'd have release please update the version number here

Reply:
though on a second look at the dart sdk, I think release please will already update version numbers in the main manifest file. Will remove this addition for now

Reply:
> So I think the utils file pulls in the [`CARGO_PKG_VERSION` env var](https://github.com/momentohq/client-sdk-rust/blob/main/src/utils.rs#L13) to use in the [`agent` header](https://github.com/momentohq/client-sdk-rust/blob/main/src/utils.rs#L104). And my understanding is that `CARGO_PKG_VERSION` gets pulled from this `.toml` file so I figured I'd have release please update the version number here

Oh I understand what you're saying now. Yes we do want it to update this so feel free to add it back in if you end up needing to.